### ISSUE TITLE
Canu skips where it ended when restart

### ISSUE THREAD

**Issue by Yutang-ETH** [marbl/canu#2104]
Hi,

I am using Canu 2.2 and here's my command:
```
canu \
    -trim-assemble \
    -p mygenome -d mygenome_canu \
    useGrid=false \
    -maxMemory=490g \
    -maxThreads=48 \
    'maxInputCoverage=100' \
    'batOptions=-eg 0.12 -sb 0.01 -dg 3 -db 3 -dr 1 -ca 500 -cp 50' \
    'correctedErrorRate=0.12' \
    genomeSize=2.8g \
    -pacbio-hifi myreads.fa 
```
Due to an outage of our machine, my Canu program was terminated. When I restarted it, I found it just skips the trimming stage to next unitigging stage and doesn't resume the unfinished overlapper step in trimming. How could I fix this?

Thank you very much.
Best wishes,
Yutang

**Comment by skoren** [comment]
Are you sure trimming wasn't already completed? What was the latest in the logs before it terminated? What is the contents of my genome.seqStore folder?

**Comment by Yutang-ETH** [comment]
Thank you very much for your quick reply. 
Yes, trimming wasn't completed because the total overlaps should be 1345, but only 900 overlaps were completed. 
The last overlap file I saw is 1050.out, however there are files missing between 900.out and 1050.out. I guess this is associated with our storage outage, which somehow made canu skip some jobs.
I also checked seqStore folder, files or the folder called version.001 in there are all created by the first day I started Canu, no recent files. 
 Is it possible to tell Canu to start from overlap 901?
